README.md

```diff
@@ -118,6 +118,8 @@ In addition to standard Axios methods (`post`, `get`, `put`, `patch`, `delete`,
 * `mockResponse` - simulates a server (web service) response
 * `mockError` - simulates a (network/server) error
 * `lastReqGet` - returns extended info about the most recent request
+* `getReqByMatchUrl` - returns extended info about the most recent request matching the given regexUrl.
+* `queue` - returns a queue with all requests received.
```

Comment:
❤️ 

Maybe there is missing a link right here...

![Screen Shot 2020-06-04 at 15 42 36](https://user-images.githubusercontent.com/7637806/83798334-7c4f1580-a67a-11ea-8b86-41889f58e826.png)

Reply:
Thanks @victorcorcos